packages/renderer/src/api/Renderer.ts

```diff
@@ -0,0 +1,7 @@
+export default interface Renderer {
+  randomPrefix: string;
+
+  renderLayouts(): void;
```

Comment:
It's not better to return a promise ? or everything in your method is synchronous ?

Reply:
in general yes, it's some sync stuff like `document.getElementById(name)!.appendChild(webComponent);`

Reply:
I can wrap it in promise for the future changes